index.html

```diff
@@ -1,229 +1,257 @@
-            style="list-style-image: url(./assets/images/diamond.png)"
-          >
-            <h1>Language</h1>
-            <ul>
-              <li><h5>Javascript</h5></li>
-              <li><h5>Ruby</h5></li>
-              <li><h5>Html</h5></li>
-              <li><h5>Css</h5></li>
-            </ul>
-          </li>
-          <li
-            class="skill"
-            style="list-style-image: url(./assets/images/rect.png)"
-          >
-            <h1>Frameworks</h1>
-            <ul>
-              <li><h5>Bootstrap</h5></li>
-              <li><h5>Ruby on Rails</h5></li>
-              <li><h5>RSPec</h5></li>
-              <li><h5>CapyBara</h5></li>
-              <li><h5>Selenium</h5></li>
-            </ul>
-          </li>
-          <li
-            class="skill"
-            style="list-style-image: url(./assets/images/circld.png)"
-          >
-            <h1>Skills</h1>
-            <ul>
-              <li><h5>Codekit</h5></li>
-              <li><h5>Github</h5></li>
-              <li><h5>Codepen</h5></li>
-              <li><h5>Gitlab</h5></li>
-              <li><h5>Terminal</h5></li>
-            </ul>
-          </li>
-        </ul>
-      </div>
-    </section>
-  </body>
-</html>
+    <hr />
+    <div class="all_skills">
+      <ul>
+        <li class="skill" style="list-style-image: url(./assets/images/diamond.png)">
+          <h1>Language</h1>
+          <ul>
+            <li>
+              <h5>Javascript</h5>
+            </li>
+            <li>
+              <h5>Ruby</h5>
+            </li>
+            <li>
+              <h5>Html</h5>
+            </li>
+            <li>
+              <h5>Css</h5>
+            </li>
+          </ul>
+        </li>
+        <li class="skill" style="list-style-image: url(./assets/images/rect.png)">
+          <h1>Frameworks</h1>
+          <ul>
+            <li>
+              <h5>Bootstrap</h5>
+            </li>
+            <li>
+              <h5>Ruby on Rails</h5>
+            </li>
+            <li>
+              <h5>RSPec</h5>
+            </li>
+            <li>
+              <h5>CapyBara</h5>
+            </li>
+            <li>
+              <h5>Selenium</h5>
+            </li>
+          </ul>
+        </li>
+        <li class="skill" style="list-style-image: url(./assets/images/circld.png)">
+          <h1>Skills</h1>
+          <ul>
+            <li>
+              <h5>Codekit</h5>
+            </li>
+            <li>
+              <h5>Github</h5>
+            </li>
+            <li>
+              <h5>Codepen</h5>
+            </li>
+            <li>
+              <h5>Gitlab</h5>
+            </li>
+            <li>
+              <h5>Terminal</h5>
+            </li>
+          </ul>
+        </li>
+      </ul>
+    </div>
+  </section>
+  <footer class="contact">
+    <h1>
+      I'm always interested in hearing about new projects, so if you'd like to
+      chat please get in touch.
+    </h1>
+    <form action="" method="post" class="contact-form">
```

Comment:
- [ ] Great job so far 👏. However, I think your `form` should have an `action`  value that is similar to  `action="https://formspree.io/f/{form_id} " `
- [ ] What you need first is to create your account at [formspree](https://formspree.io/html/) and then generate the `form_id`. 
- [ ] In your `action` value replace `{form_id}` with the `id` you generated from  [formspree](https://formspree.io/html/). 👌

- [ ] Lastly, please make that a user can successfully submit a `form`. After that is done you can proudly submit it for review. 🤗